app/services/DateService.scala

```diff
@@ -39,23 +36,23 @@ class DateService @Inject()(implicit val frontendAppConfig: FrontendAppConfig) e
   }
 
   // with respect to the current calendar year
-  private def TAX_YEAR_LAST_DAY: LocalDate = LocalDate.of(getCurrentDate.getYear, APRIL, 6)
+  private def TAX_YEAR_LAST_DAY(isTimeMachineEnabled: Boolean): LocalDate = LocalDate.of(getCurrentDate(isTimeMachineEnabled).getYear, APRIL, 6)
```

Comment:
method naming convention: https://docs.scala-lang.org/style/naming-conventions.html
should start from lower camel case